pipelines/jobs/configurations/jdk20_pipeline_config.groovy

```diff
@@ -164,7 +164,7 @@ class Config20 {
                         'temurin'   : '--create-jre-image --create-sbom'
                 ],
                 test                : [
-                        nightly: ['sanity.openjdk'],
+                        // nightly: ['sanity.openjdk'],
```

Comment:
prefer not to have commented out code...

Reply:
Fixed.